sdk/search/search-documents/test/utils/setup.ts

```diff
@@ -628,3 +633,7 @@ export async function createSimpleIndex(client: SearchIndexClient, name: string)
   };
   await client.createIndex(index);
 }
+
+export function createRandomIndexName(): string {
+  return `hotel-live-test${Math.floor(Math.random() * 1000) + 1}`;
```

Comment:
Since index names can be up to 128 characters, we could generate a uuid for this if you wanted to be more sure of avoiding collisions.

Reply:
I will keep that in mind. For now, since it is only 4 machines, this should be fine.